opts/envfile.go

```diff
@@ -4,9 +4,16 @@ import (
 	"bufio"
 	"fmt"
 	"os"
+	"regexp"
 	"strings"
 )
 
+var (
+	// Variables set by the user must have a name consisting solely of
+	// alphabetics, numerics, and underscores - the first of which must not be numeric.
+	EnvironmentVariableRegexp = regexp.MustCompile("^[[:alpha:]_][[:alpha:][:digit:]_]+$")
```

Comment:
I think the "+" needs to be "*" otherwise we won't support single-char env names.